Index: .idea/misc.xml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+><?xml version=\"1.0\" encoding=\"UTF-8\"?>\n<project version=\"4\">\n  <component name=\"Black\">\n    <option name=\"sdkName\" value=\"Python 3.11 (Proyecto-2025-TdV-Grupo-D)\" />\n  </component>\n  <component name=\"ProjectRootManager\" version=\"2\" project-jdk-name=\"Python 3.11 (Proyecto-2025-TdV-Grupo-D)\" project-jdk-type=\"Python SDK\" />\n</project>
===================================================================
diff --git a/.idea/misc.xml b/.idea/misc.xml
--- a/.idea/misc.xml	(revision e627fd3aa8f0279008a446483e28fafeff35cf81)
+++ b/.idea/misc.xml	(date 1746546863658)
@@ -3,5 +3,5 @@
   <component name="Black">
     <option name="sdkName" value="Python 3.11 (Proyecto-2025-TdV-Grupo-D)" />
   </component>
-  <component name="ProjectRootManager" version="2" project-jdk-name="Python 3.11 (Proyecto-2025-TdV-Grupo-D)" project-jdk-type="Python SDK" />
+  <component name="ProjectRootManager" version="2" project-jdk-name="Python 3.10 (Proyecto-2025-TdV-Grupo-D)" project-jdk-type="Python SDK" />
 </project>
\ No newline at end of file
Index: .idea/Proyecto-2025-TdV-Grupo-D.iml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+><?xml version=\"1.0\" encoding=\"UTF-8\"?>\n<module type=\"PYTHON_MODULE\" version=\"4\">\n  <component name=\"NewModuleRootManager\">\n    <content url=\"file://$MODULE_DIR$\">\n      <excludeFolder url=\"file://$MODULE_DIR$/.venv\" />\n    </content>\n    <orderEntry type=\"jdk\" jdkName=\"Python 3.11 (Proyecto-2025-TdV-Grupo-D)\" jdkType=\"Python SDK\" />\n    <orderEntry type=\"sourceFolder\" forTests=\"false\" />\n  </component>\n  <component name=\"PyDocumentationSettings\">\n    <option name=\"format\" value=\"PLAIN\" />\n    <option name=\"myDocStringFormat\" value=\"Plain\" />\n  </component>\n</module>
===================================================================
diff --git a/.idea/Proyecto-2025-TdV-Grupo-D.iml b/.idea/Proyecto-2025-TdV-Grupo-D.iml
--- a/.idea/Proyecto-2025-TdV-Grupo-D.iml	(revision e627fd3aa8f0279008a446483e28fafeff35cf81)
+++ b/.idea/Proyecto-2025-TdV-Grupo-D.iml	(date 1746546863656)
@@ -4,7 +4,7 @@
     <content url="file://$MODULE_DIR$">
       <excludeFolder url="file://$MODULE_DIR$/.venv" />
     </content>
-    <orderEntry type="jdk" jdkName="Python 3.11 (Proyecto-2025-TdV-Grupo-D)" jdkType="Python SDK" />
+    <orderEntry type="jdk" jdkName="Python 3.10 (Proyecto-2025-TdV-Grupo-D)" jdkType="Python SDK" />
     <orderEntry type="sourceFolder" forTests="false" />
   </component>
   <component name="PyDocumentationSettings">
